messages/resume_scratch.md

```diff
@@ -4,9 +4,11 @@ Resume the creation of an incomplete scratch org.
 
 # description
 
-When the original "<%= config.bin %> org create scratch" command either times out or is run with the --async flag, it displays a job ID.
+When the original "<%= config.bin %> org create scratch" command either times out or is run with the --async flag, it
+displays a job ID.
```

Comment:
@WillieRuemmele , can you revert this message back to its original, without the extra newline?  The newline is showing up in the --help. Thx!